content/news/buzz/august-2023-status-update/contents.lr

```diff
@@ -0,0 +1,53 @@
+
+* The Toga Audit is complete on macOS, iOS and GTK! The last pieces were the audit of `Window <https://github.com/beeware/toga/pull/2058>`__ and `App <https://github.com/beeware/toga/pull/2075>`__; with those PRs (and some others that are outstanding), we have 100% code coverage for Toga's codebase. Windows and Android are catching up rapidly - we've completed the audit of `OptionContainer <https://github.com/beeware/toga/pull/1996>`__, `Table <https://github.com/beeware/toga/pull/2011>`__, and `Tree <https://github.com/beeware/toga/pull/2017>`__, as well as a major revamp of `how layouts are calculated <https://github.com/beeware/toga/pull/2020>`__, which ensures that minimum window sizes are enforced, and layouts on High DPI displays are correct.
+
+* We added a `Texual backend for Toga <https://github.com/beeware/toga/pull/2065>`__. This means the same Toga application will now run on 3 desktop platforms, 2 mobile platforms, as a single page web app, and in a terminal window. As with the web backend, there's still a lot of work to fill out all the widgets, but we've got enough in place to run the BeeWare tutorial app and the first 2 Toga tutorial steps.
+
+* We `improved the architecture and word size checks that Briefcase performs when building apps <https://github.com/beeware/briefcase/pull/1360>`__. This effectively adds support for 32-bit x86 and ARM platforms; we're limited in our ability to perform automatic tests to verify these platforms work as advertised, and upstream tooling isn't always up to the task (for example, Linuxdeploy doesn't support ARM at any word size) - but Briefcase now performs much more robust checks, and will make a best effort when support *should* exist.
+
+* We added `support for openSUSE (and other SUSE-derived distributions) <https://github.com/beeware/briefcase/pull/1417>`__ to Briefcase.
+
+* We `updated the version of the Android command line tools that Briefcase uses <https://github.com/beeware/briefcase/pull/1397>`__.
+
+* We resolved a long standing issue with the use of `Objective C blocks on M1 hardware <https://github.com/beeware/rubicon-objc/pull/357>`__.
+
+What's next?
+--------------
+
+Based on current progress, it seems highly likely that the Toga audit will be complete on Windows and Android by the end of September. This will bring the Toga audit to completion, providing a very stable foundation for future work on Toga. It also means you can expect a Toga v0.3.2 release in the coming month, incorporating the myriad bug fixes and improvements that the audit has revealed. This release will include a number of backwards-incompatible changes identified as part of the audit, so be prepared for a bumpier-than-usual upgrade process.
```

Comment:
Given the large number of backward incompatible changes and substantial improvements, should we call the next version 0.4.0?

Reply:
I've had a similar thought. It's a little weird that Toga spend almost 5 years as a pre-release 0.3, then less than 6 months later there's a 0.4... but I agree that a version bump is likely called for. 

For now, I'll update the blog post to say *a* release.